package.json

```diff
@@ -24,7 +24,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint"
+      "pre-push": "yarn lint && gradle checkstyleMain checkstyleTest checkstyleIntegration"
```

Comment:
minor: should be in different commit

Reply:
forgot to remove that from PR, good spot